ts/screens/wallet/payment/ScanQrCodeScreen.tsx

```diff
@@ -220,6 +232,28 @@ class ScanQrCodeScreen extends React.Component<Props, State> {
     }
   }
 
+  public async componentDidMount() {
+    if (Platform.OS !== "android") {
+      return;
+    }
+    const hasPermission = await PermissionsAndroid.check(
```

Comment:
It is just a warning/reminder: we display the rational also when the user chose "never ask again"
But we can't know his/her preference without asking the permission